forwards_paper.tex

```diff
@@ -780,9 +793,6 @@ \subsubsection*{Sequential simplification and prior history}
 25\%, with the lowest run time occurring when simplifying every $10^3$ generations.
 Thus, there is a memory-versus-speed tradeoff
 -- simplifying more often would keep fewer extinct nodes and edges in memory.
-The run-time costs associated
```

Comment:
I don't think this is correct; the overheads are pretty minimal, and running simplify does take a significant chunk of time.